client/client_test.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestTimeout(t *testing.T) {
+	clientConfig := Configuration{
+		Endpoint: "http://httpbin.org/delay/2?",
+		Timeout:  1 * time.Second,
+	}
+	apiClient := NewClient(clientConfig)
+	_, _, err := apiClient.GetUserOrganizations("foo", "foo", "foo", "foo")
+	if err == nil || !strings.Contains(err.Error(), "Client.Timeout exceeded") {
```

Comment:
When it is a URL error, we can check if it is an timeout error. See https://golang.org/pkg/net/url/#Error.Timeout.